gui/src/renderer/components/ParameterInput.js

```diff
@@ -1,12 +1,45 @@
 import * as React from 'react'
 import styled from '@emotion/styled'
 
-import { Box, Checkbox, FormControl, MenuItem, Select, Stack, Tooltip, Typography, TextField } from '@mui/material'
+import { Box, Chip, Button, Checkbox, FormControl, MenuItem, Select, Stack, Tooltip, Typography, TextField } from '@mui/material'
 
 const StyledCheckbox = styled(Checkbox)(({ theme }) => ({
     padding: 0.5
 }))
 
+const SingleFolderSelection = ({parameter, onChange = () => {}}) => {
+
+    const handleSelect = () => {
+        window.electronAPI.getSingleFolder().then((folder) => {
+            onChange(folder);
+        }).catch((err) => {
+            console.log(err);
+        })
+    }
+
+    const folderName = parameter.replace(/^.*[\\\/]/, '')
```

Comment:
why do we need this?

Reply:
For the sake of simplicity, every parameter input receives the value as its stored in the config. For file or folder inputs we want to display a small "chip" to indicate that a file has been selected (Grey bubbles #134). This regex selects only the file or folder name for display.